test/Token.js

```diff
@@ -0,0 +1,28 @@
+const { expect } = require("chai");
+
+describe("Token contract", () => {
+  let tokenContract;
+  let owner;
+  let addr1;
+  let addr2;
+  let addrs;
+
+
+  beforeEach(async () => {
```

Comment:
p sure this should be beforeAll so we don't redeploy before every test

Reply:
before() not beforeAll()

Reply:
We want a clean slate after every test so that tests do not cause side effects on other tests and only test one piece of functionality.